.travis.yml

```diff
@@ -1,43 +1,29 @@
-dist: trusty
-sudo: false
-
 language: python
 
+env:
+  global:
+    - CIBW_SKIP="cp34-* cp35-*"
+    - CIBW_TEST_REQUIRES="tox"
+    - CIBW_TEST_COMMAND="tox --installpkg wheelhouse/*.whl --workdir {project} -c {project}"
```

Comment:
The —installpkg option needs to get the full path to the single wheel. You need to evaluate the glob pattern in a subshell.. it’s getting too complicated. Probably not worth it.